platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/reconnect/emergency/EmergencyReconnectProtocol.java

```diff
@@ -78,7 +82,8 @@ public EmergencyReconnectProtocol(
             final SignedStateFinder stateFinder,
             final int reconnectSocketTimeout,
             final ReconnectMetrics reconnectMetrics,
-            final ReconnectController reconnectController) {
+            final ReconnectController reconnectController,
```

Comment:
null annotations and checks are missing

Reply:
Annotated with `@NonNull` + a check is added.